pages/api/prescriptions/read.ts

```diff
@@ -22,12 +28,18 @@ export default async function handler(
         prescription = await prisma.prescription.findMany({
           where: {
             patientId: patientId
+          },
+          include: {
+            Location: true,
+            LockerBox: true
```

Comment:
these two calls should really be two separate endpoints. one should be on the user route since we are getting the "users" or the "patients" prescriptions. the `api/prescriptions` endpoints should be when we want to get all prescriptions but like to get a specific users I think It could be

`GET /api/patient/[id]/presciptions`